src/routes/write-exercise/components/Feedback.tsx

```diff
@@ -66,9 +70,7 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
   return (
     <Background source={background} testID='background-image'>
       <Icon width={28} height={28} />
-      <StyledText numberOfLines={3} ellipsizeMode='tail'>
-        {message}
-      </StyledText>
+      <StyledText>{message}</StyledText>
```

Comment:
These changes are from your other branch LUN-418 and should not appear here. This happens if you do `git checkout -b LUN-417...` when being on LUN-418 branch. You always need to do `git checkout main` before checking out a new branch.
To fix this you normally need to change to base branch see here: https://stackoverflow.com/questions/10853935/change-branch-base
But in this case, you can just merge your LUN-418 branch and it is just fine.